examples/forecasts/example_forecasting_task.py

```diff
@@ -0,0 +1,26 @@
+from forecasting.task.forecasting.forecasting import Forecasting as task
+
+from forecasting.task.forecasting.dataset import TimeSeriesDataset
+import core as ag
+import matplotlib.pyplot as plt
+
+dataset = TimeSeriesDataset(
+    train_path="./COV19/processed_train.csv",
+    test_path="./COV19/processed_test.csv",
+    prediction_length=19,
+    index_column="name",
+    target_column="ConfirmedCases",
+    time_column="Date")
+
+print(dataset.train_data, dataset.test_data)
+metric = "MAPE"
+predictor = task.fit(train_data=dataset.train_data,
+                     test_data=dataset.test_data,
```

Comment:
I thought we agreed not to call this argument `test_data` as it's improper to tune hyperparameters on the test-data. Can we call it `tuning_data` or `validation_data` instead?